adops/line-item-creation.md

```diff
@@ -106,10 +106,14 @@ Prebid supports many media types, and you can set up a single line item with mul
 
 Grouping media types within line items is typically dictated by the pricing structure:
 
-- Banner, outstream video, and native generally run together because they have similar pricing expectations and therefore can share [price granularities](/adops/price-granularity.html).
+- Banner and native generally run together because they have similar pricing expectations and therefore can share [price granularities](/adops/price-granularity.html).
 - Instream video is normally created as a separate set of line items because they are usually priced higher than other formats, often requiring custom price granularity.
+- Outstream video needs to run with a set of line items that will support appropriate rendering. For Prebid.js, outstream is delivered along with banner, while for Prebid Mobile, outstream goes with instream video.
```

Comment:
Does Prebid Mobile have a player capable of parsing vast tags ? That's all you need for outstream in a browser, it shouldn't have to be delivered with banner

Reply:
Prebid SDK uses the google sdk to render outstream. It, in turn, requires that the GAM creative be directly VAST, which conflicts with how rendering needs to happen in Prebid.js